### PR TITLE
Smoke: Separated scripts for DialNumber and PerformAudioPassThru from smoke API script

### PR DESCRIPTION
**Pull request by GetmanetsIrina**
Separated scripts for DialNumber and PerformAudioPassThru from smoke API script

### CODE REVIEW

File: test_scripts/Smoke/API/031_PerformAudioPassThru_PositiveCase.lua [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+  requestUiParams = requestUiParams,
+  requestTtsParams = requestTtsParams
+}
+
+--[[ Local Functions ]]
+local function file_check(file_name)
+  local file_found = io.open(file_name, "r")
+  if nil == file_found then
+    return false
+  else
```
Comment (aderiabin) [inline test_scripts/Smoke/API/031_PerformAudioPassThru_PositiveCase.lua]: @GetmanetsIrina Else statement can be omitted here.

```
local function file_check(file_name)
  local file_found = io.open(file_name, "r")
  if nil == file_found then
     return false
  end
  return true
end
```

File: test_scripts/Smoke/API/031_PerformAudioPassThru_PositiveCase.lua [hunk 2]
```diff
@@ -0,0 +1,166 @@
...
+    expectOnHMIStatusWithAudioStateChanged_PerformAudioPassThru(self)
+    self.mobileSession1:ExpectNotification("OnAudioPassThru")
+    :Times(1)
+    :Timeout(11000)
+    self.mobileSession1:ExpectResponse(cid, { success = true, resultCode = "SUCCESS" })
+    :ValidIf (function()
+        local file = commonPreconditions:GetPathToSDL() .. "storage/" .. "audio.wav"
+        if file_check(file) ~= true then
+          return false, "Can not found file: audio.wav"
+        else
```
Comment (aderiabin) [inline test_scripts/Smoke/API/031_PerformAudioPassThru_PositiveCase.lua]: @GetmanetsIrina Else statement can be omitted here.

File: test_scripts/Smoke/API/031_PerformAudioPassThru_PositiveCase.lua [hunk 3]
```diff
@@ -0,0 +1,166 @@
...
+local function file_check(file_name)
+  local file_found = io.open(file_name, "r")
+  if nil == file_found then
+    return false
+  else
+    return true
+  end
+end
+
+local function expectOnHMIStatusWithAudioStateChanged_PerformAudioPassThru(self)
```
Comment (dboltovskyi) [inline test_scripts/Smoke/API/031_PerformAudioPassThru_PositiveCase.lua]: @GetmanetsIrina There is function with the same name in common module. Is it possible to use it here?

File: test_scripts/Smoke/API/031_PerformAudioPassThru_PositiveCase.lua [hunk 4]
```diff
@@ -0,0 +1,166 @@
...
+        local function uiResponse()
+          self.hmiConnection:SendResponse(data.id, data.method, "SUCCESS", {})
+          sendOnSystemContext(self, "MAIN", pParams.requestUiParams.appID)
+        end
+        RUN_AFTER(uiResponse, 1500)
+      end)
+    expectOnHMIStatusWithAudioStateChanged_PerformAudioPassThru(self)
+    self.mobileSession1:ExpectNotification("OnAudioPassThru")
+    :Times(1)
+    :Timeout(11000)
```
Comment (dboltovskyi) [inline test_scripts/Smoke/API/031_PerformAudioPassThru_PositiveCase.lua]: @GetmanetsIrina Do we really need non-default Times() and Timeout() here?

File: test_scripts/Smoke/API/031_PerformAudioPassThru_PositiveCase.lua (hunk 1 above)
Comment (GetmanetsIrina) [reply test_scripts/Smoke/API/031_PerformAudioPassThru_PositiveCase.lua]: @aderiabin , Done in fb183773759e98c5603d803a4be087a44dba7ae6.

File: test_scripts/Smoke/API/031_PerformAudioPassThru_PositiveCase.lua (hunk 2 above)
Comment (GetmanetsIrina) [reply test_scripts/Smoke/API/031_PerformAudioPassThru_PositiveCase.lua]: @aderiabin , Done in fb183773759e98c5603d803a4be087a44dba7ae6.

File: test_scripts/Smoke/API/031_PerformAudioPassThru_PositiveCase.lua (hunk 3 above)
Comment (GetmanetsIrina) [reply test_scripts/Smoke/API/031_PerformAudioPassThru_PositiveCase.lua]: @dboltovskyi , according to decision about covering only media app and creation separated scrips for non-media application function was removed and added expectation for media app in performAudioPassThru.

File: test_scripts/Smoke/API/031_PerformAudioPassThru_PositiveCase.lua (hunk 4 above)
Comment (GetmanetsIrina) [reply test_scripts/Smoke/API/031_PerformAudioPassThru_PositiveCase.lua]: Items are redundant, removed.

**Pull request merged**